hw/ip/pattgen/data/pattgen.prj.hjson

```diff
@@ -12,7 +12,7 @@
         version:            "1.0",
         life_stage:         "L1",
         design_stage:       "D1",
-        verification_stage: "V1",
+        verification_stage: "V2",
```

Comment:
Please move this to a separate PR once all checklist items are marked done.

Reply:
Roll back V1 for review